.pre-commit/run_buf.sh

```diff
@@ -7,4 +7,4 @@ fi
 
 buf generate
 buf lint
-buf breaking --against '.git#branch=origin/main'
```

Comment:
disabled this since these are breaking changes, but the protos are still in draft, so this breaking changes are ok.

Reply:
don't forget to uncomment this later